SpatialGDK/Source/SpatialGDK/Private/Interop/GlobalStateManager.cpp

```diff
@@ -455,10 +456,6 @@ void UGlobalStateManager::QueryGSM(const QueryDelegate& Callback)
 		}
 		else
 		{
-			if (NetDriver->VirtualWorkerTranslator.IsValid())
```

Comment:
Ironically, the translation was originally part of the GSM and when we moved it to its own entity, we forgot some of the original code, so this framework was still here.